### PR TITLE
Add target blank to external pages and do not load them as iframe

### PR DESCRIPTION
**Pull request by zero-24**
Make sure external links are open in a new tab as reported here: https://github.com/joomla/joomla-websites/issues/1631#issuecomment-854869966 by @brianteeman 

cc @wilsonge @HLeithner

### CODE REVIEW

File: templates/base.html.twig [hunk 1]
```diff
@@ -35,10 +35,10 @@
                         <div class="span5">
                             <div class="btn-toolbar pull-right">
                                 <div class="btn-group">
-                                    <a href="https://downloads.joomla.org" class="btn btn-large btn-warning">Download</a>
+                                    <a href="https://downloads.joomla.org" class="btn btn-large btn-warning"  target="_blank">Download</a>
```
Comment (HLeithner) [inline templates/base.html.twig:38]: Double space

Comment (zero-24) [reply templates/base.html.twig:38]:
```suggestion
                                    <a href="https://downloads.joomla.org" class="btn btn-large btn-warning" target="_blank">Download</a>
```

Comment (zero-24) [reply templates/base.html.twig]: Fixed thanks.

**Comment by HLeithner** [comment]
Thanks

**Comment by zero-24** [comment]
Thanks the errors are gone and open in a new window now. 👍

**Comment by brianteeman** [comment]
Confirming the fix - thanks

**Pull request merged**